### PR TITLE
Create review show page with ratings

### PR DESCRIPTION
**Pull request by hnlee**
Make models for rating and for a rating check. (A rating check would be, for
example, saying that a review is kind. By making it a model object, we allow for
extensibility in the future, i.e. if we want to add the option to evaluate a
review on additional criteria other than kind, specific, and actionable.)

Make routes for review show page. Make controller for reviews to display the ratings
associated with a review and the rating checks associated with each rating.

Make a view for review show page, which should display the content of the
review, followed by the ratings associated with it.

Also include pry in Gemfile due to its help in debugging.

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class ReviewsController < ApplicationController
+  def show
+   @review = Review.find_by_id(params[:id])
```
Comment (JPhoenx) [inline app/controllers/reviews_controller.rb]: This spacing is off by a character

Comment (JPhoenx) [inline app/controllers/reviews_controller.rb]: Also... we really need to use the presenter pattern here.

File: db/migrate/20161020164549_create_ratings.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateRatings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :ratings do |t|
```
Comment (JPhoenx) [inline db/migrate/20161020164549_create_ratings.rb]: We pulled these out into a RatingCheck class correct?

File: app/controllers/reviews_controller.rb (hunk 1 above)
Comment (hnlee) [inline app/controllers/reviews_controller.rb]: 👍

**Comment by NicoleCarpenter** [comment]
👍

**Comment by NicoleCarpenter** [comment]
@JPhoenx you can review and pull if you're happy

**Pull request merged**